sandbox/pages/SandboxModals.vue

```diff
@@ -0,0 +1,600 @@
+            @proceed="data.modalVisible = false"
+          >
+            When no title is provided and hideCloseIcon is true, the modal header is hidden.
+          </KModal>
+        </KComponent>
+      </SandboxSectionComponent>
+      <SandboxSectionComponent
+        description="Only available in KModal."
+        title="hideCancelButton"
+      >
+        <KComponent
+          v-slot="{ data }"
+          :data="{ modalVisible: false }"
+        >
+          <div class="horizontal-container">
+            <KButton
+              @click="data.modalVisible = !data.modalVisible"
+            >
+              KModal
+            </KButton>
+          </div>
+
+          <KModal
+            hide-cancel-button
+            title="KModal"
+            :visible="data.modalVisible"
+            @cancel="data.modalVisible = false"
+            @proceed="data.modalVisible = false"
+          />
+        </KComponent>
+      </SandboxSectionComponent>
+      <SandboxSectionComponent
+        description="Only available in KPrompt."
+        title="message & confirmationText & confirmationPrompt"
+      >
+        <KComponent
+          v-slot="{ data }"
+          :data="{ promptVisible: false }"
+        >
+          <KButton
+            @click="data.promptVisible = !data.promptVisible"
+          >
+            KPrompt
+          </KButton>
+
+          <KPrompt
+            confirmation-prompt="KPrompt confirmation prompt: type {confirmationText} to confirm."
+            confirmation-text="confirm"
+            message="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+            title="KPrompt"
+            :visible="data.promptVisible"
+            @cancel="data.promptVisible = false"
+            @proceed="data.promptVisible = false"
+          />
+        </KComponent>
+      </SandboxSectionComponent>
+      <SandboxSectionComponent title="actionButtonText & actionButtonAppearance & actionButtonDisabled">
+        <KComponent
+          v-slot="{ data }"
+          :data="{ modalVisible: false, promptVisible: false, modalActionEnabled: true, promptActionEnabled: true }"
+        >
+          <div class="horizontal-container">
+            <div class="vertical-container">
+              <div>
+                <KButton
+                  @click="data.modalVisible = !data.modalVisible"
+                >
+                  KModal
+                </KButton>
+              </div>
+              <KInputSwitch
+                v-model="data.modalActionEnabled"
+                label="Action button enabled"
+              />
```

Comment:
These input switches in the sandbox examples; can't you just combine the two examples in the section to 1 instance of the KInputSwitch instead of having 1 for each? Applies to each section
![image](https://github.com/Kong/kongponents/assets/2229946/689c54bb-9ac8-4cbf-8aaf-e2f97a24739a)

Reply:
Actually I would prefer to keep those toggles separate for each component. Having one for each helps making sure that attributes don't leak since KPrompt is using KModal under the hood.